README.md

```diff
@@ -50,6 +50,11 @@ cd src
 make
 cd ..
 ```
+or with cmake:
+```
+cmake src -B lib -DCMAKE_BUILD_TYPE=Release
```

Comment:
It is great to see some modern CMake here, but especially for docs I would make all args explicit.
```suggestion
cmake -S src -B lib -DCMAKE_BUILD_TYPE=Release
```